codegen/scala-gen/src/main/scala/kalix/codegen/scalasdk/impl/SourceGenerator.scala

```diff
@@ -139,7 +139,7 @@ object SourceGenerator {
       protoFileName = "",
       name = "",
       protoPackage = packageString,
-      javaPackageOption = None,
+      javaPackageOption = Some(packageString),
```

Comment:
I know I did this myself, but looking at it again with fresh eyes it doesn't necessarily make sense to me anymore: if we initialize it with `Some`, does it need to be an `Option` at all?

Reply:
I think we should keep it. Or change in another PR in any case. 

As far as I can tell, when we fill it from a descriptor, it can be empty: 
https://github.com/lightbend/kalix-jvm-sdk/blob/c5c537144006d425a3087b28aa9a772dfb0376fe/codegen/core/src/main/scala/kalix/codegen/MessageType.scala#L177-L178